test/edu-benefits/1990e/config/benefitSelection.unit.spec.jsx

```diff
@@ -16,9 +16,8 @@ describe('Edu 1990e benefitEligibility', () => {
           data={{}}
           uiSchema={uiSchema}/>
     );
-    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
-
-    expect(fields.length).to.equal(2);
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input'))
```

Comment:
Originally yes, I changed it because of a failure, but after running npm install and getting updated, I noticed that in some tests we just check that inputs are present and in others we test for exact counts. Glad to go with exact counts if you don't think these will be changing often.